MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -908,10 +923,22 @@ protected override string GetVideoArguments(StreamState state, EncodingOptions e
             }
             else
             {
-                var keyFrameArg = string.Format(
+                var keyFrameArg = string.Format(CultureInfo.InvariantCulture,
```

Comment:
All parameters should be on a new line.
```suggestion
                var keyFrameArg = string.Format(
                    CultureInfo.InvariantCulture,
```